roles/StackStorm.st2/tasks/main.yml

```diff
@@ -1,4 +1,54 @@
 ---
+- name: Install python3 SELinux policies
+  become: yes
+  yum:
+    name: libselinux-python3
```

Comment:
Do we really need `libselinux-python3` dependency in Ansible deployment? I can't find any usage for `st2` role.

Reply:
It depends really on what python3 modules are used, ST2 itself doesn't need it, but there may be packs that import modules that need it given they are being moved from python2 -> 3, and therefore don't get the python3 selinux policies by default.

We could remove it, and then it would be up to the packs to ensure that they add the dependency if they need it.

I'm fine with either method.

Reply:
That dependency is already installed in `nginx` role:
https://github.com/StackStorm/ansible-st2/blob/75193b9bac1c36ef1a4ad15b5cf6433e01377c09/roles/StackStorm.nginx/vars/redhat_8.yml#L1-L3

which is then used to adjust some security restrictions that may come by default in RHEL/CentOS
https://github.com/StackStorm/ansible-st2/blob/75193b9bac1c36ef1a4ad15b5cf6433e01377c09/roles/StackStorm.nginx/tasks/nginx_redhat.yml#L81-L95


With that, we don't need it in `st2` role, but may need to update the https://github.com/StackStorm/ansible-st2/blob/master/roles/StackStorm.nginx/vars/redhat.yml

Reply:
Thanks - that's a better place to put it. Will move it to Nginx role.